crates/utils/re_video/src/demux/mod.rs

```diff
@@ -509,6 +507,43 @@ impl VideoDataDescription {
         self.samples.num_elements()
     }
 
+    /// Duration of the video.
+    ///
+    /// For videos that may still grow this is an estimate of the video length so far.
+    /// (we don't know for sure how long the last sample is going to be)
+    pub fn duration(&self) -> Option<std::time::Duration> {
```

Comment:
If it falls back to an estimate, why can it return None?

Reply:
because we may not know the timescale or there may be only a single stream sample. That's a weird corner case for streams on non temporal timelines. Originally we just gave them a made up timescale, but the both of us settled on having it `None`.

Adding this to docs